lib/config.js

```diff
@@ -389,12 +388,11 @@ var parseConfig = function (configFilePath, cliOptions) {
       return process.exit(1)
     }
   } else {
-    log.debug('No config file specified.')
     // if no config file path is passed, we define a dummy config module.
     configModule = function () {}
   }
 
-  var config = new Config()
+    var config = new Config()
```

Comment:
this seems unrelated/wrong indentation

Reply:
Not quite sure how that happened, I've edited it via the Github editor.